AUTHORS.md

```diff
@@ -72,6 +72,8 @@ To contributors: please add your name to the list when you submit a patch to the
 * **[Dan Ciborowski](https://github.com/dciborow)**
    * ALS operationalization notebook
    * SAR PySpark improvement
+* **[David Davó](https://github.com/daviddavo)**
```

Comment:
@daviddavo could you please move it below Daniel Schneider so it follow the alphabetical order?

Reply:
Sorry, I read it wrong